intg-denonavr/driver.py

```diff
@@ -362,6 +366,7 @@ async def main():
     for device in config.devices.all():
         _configure_new_avr(device, connect=False)
 
+    # TODO: is this still needed for telnet?
```

Comment:
This should be kept, even when using Telnet. If Telnet is somehow no longer working and no commands are sent to trigger an update, then the poller will get AVR updates if the user uses another method to control the AVR (for example with some HA automatation or an additional (IR-)remote).